src/components/Widget/index.tsx

```diff
@@ -33,6 +33,7 @@ export default function Widget({ defaultToken }: WidgetProps) {
       <SwapWidget
         hideConnectionUI
         jsonRpcUrlMap={RPC_URLS}
+        disableBranding
```

Comment:
nit: put this just under `hideConnectionUI` so that the boolean attributes are all in one spot.